docs/make_work.jl

```diff
@@ -104,6 +109,14 @@ end
 
 function doit(Oscar::Module; strict::Bool = true, local_build::Bool = false, doctest::Union{Bool,Symbol} = true)
 
+  # Remove symbolic links from earlier runs
```

Comment:
I'm not sure I see your point? I was thinking whether one should clean up before or after the build and doing it before seemed a bit more reliable, but I can't say I have a strong opinion.

Reply:
Am I getting the order of things wrong? I thought once you run `build_doc` it would now remove the symlinks and create new ones. Or is it already stumbling on the issue with the broken symlinks before?

Reply:
@joschmitt you wrote:
> I just switched branches and it errors because some directory doesn't exist anymore.

There are (at least) two different errors you might refer to here, and I probably guessed wrongly
1. you switch successfully from one branch to another and try to build docs, but it fails due to a bad symlink being present; your patch here will deal with that because it clears it before switching
2. switching could fail because something that's a symlink now used to be an actual file, and switching won't work because the symlink is in the way. To solve that we need to also clean symlinks *after* building the documentation. That's the kind of problem I was thinking about. One may argue that it is not so important because it will go away over time anyway (unless someone tries to `git bisect` a regression).